modules/effects/arrow-effects/src/main/kotlin/arrow/effects/typeclasses/Bracket.kt

```diff
@@ -44,6 +81,33 @@ interface Bracket<F, E> : MonadError<F, E> {
    *
    * @param release is the action that's supposed to release the allocated resource after `use` is done, irregardless
    * of its exit condition.
+   *
+   * {: data-executable='true'}
+   *
+   * ```kotlin:ank
+   * import arrow.effects.IO
+   *
+   * class File(url: String) {
```

Comment:
What happens with this? In the screenshot, it's all gone.

Reply:
It shows everything collapsed except for what is in between `//sampleStart` and `//sampleEnd`